mcy/src/components/Layout/Drawer.jsx

```diff
@@ -1,5 +1,127 @@
+import * as React from "react";
+import { useNavigate } from "react-router-dom";
+import { Stack, styled } from "@mui/material";
+import Box from "@mui/material/Box";
```

Comment:
```suggestion
import { Stack, styled } from "@mui/material";
import Box from "@mui/material/Box";
```
import 방식은 통일하도록 가능하면 아래처럼 import 하면 나중에 빌드했을 때 트리쉐이킹이 일어나서 번들 사이즈가 줄어들어서 좋음 그럼 `import Stack from "@mui/material/Stack";` 이런식으로 바꾸면 되겠지?

@dony0720 도 참고